Woof/Woof/ViewModel/LoginViewModel.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+/// An interface that provides interaction between the `LoginView` and the model layer.
+final class LoginViewModel {
+    /// Requests the model layer to save the selected user role.
```

Comment:
nit:

You don't have to describe implementation details:

```suggestion
    /// Stores the selected user role.
```

or

```suggestion
    /// Sets the current's user role in the app.
```